runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -482,6 +483,43 @@ public void onChatMessage(ChatMessage event)
 		}
 	}
 
+	@Subscribe
+	public void onLocalPlayerDeath(LocalPlayerDeath death)
+	{
+		removeGameTimer(ANTIDOTEPLUS);
```

Comment:
Add these to a collection and iterate the collection rather than using 100 lines of code.

Honestly I'm not sure this is the correct way to go about this; Future proofing will be a nightmare, as people will create new timers which could be cleared on death, but don't know/think to add to, this class.